app/readings/NoteReadings.module.css

```
@@ -0,0 +1,561 @@
+.header {
+    margin-bottom: 2rem;
+    padding-bottom: 1.5rem;
+    border-bottom: 1px solid var(--border-color);
+}
+
+.title {
+    font-size: 2rem;
+    font-weight: 700;
+    color: var(--text-color);
+    margin-bottom: 0.5rem;
+    font-family: 'Inter', -apple-system, BlinkMacSystemFont, sans-serif;
```

Stylelint (17.10.0)

[error] Expected no quotes around "Inter" (font-family-name-quotes)

(font-family-name-quotes)